src/robusta/core/model/base_params.py

```diff
@@ -83,6 +83,10 @@ class ResourceInfo(BaseModel):
 class HolmesParams(ActionParams):
     holmes_url: Optional[str]
     model: Optional[str]
+    # Additional internal context that helps runner to send investigation to appropriate sinks
+    # for now it is used only for passing thread_ts to slack sink internally;
+    robusta_context: Optional[Dict[str, Any]] = None 
```

Comment:
Thinking here loud, we already have SlackCallbackParams which being used in other flows. I think it will be good idea to add relevant info there and reuse in relevant flows.
WDYT?